docs/data_structures/calc_status.rst

```diff
@@ -0,0 +1,74 @@
+.. _datastruct-calc-status:
+
+calc_status
+===========
+
+The ``calc_status`` is an integer attribute with a corresponding description that can be used in :ref:`sim_f<api_sim_f>` or :ref:`gen_f<api_gen_f>` functions to capture the status of a calcuation. This is returned to the manager and printed to the ``libE_stats.txt`` file. This is not currently used by the manager, but can still provide a useful summary in libE_stats.txt. The user determines the status of the calculation, as it could include multiple application runs. It can be added as a third return variable in sim_f or gen_f functions.
```

Comment:
We say "This is not currently used by the manager" but isn't it used when a worker stops being persistent (on it's own)?

Reply:
Ok, in that case I will also move these into the calc_status section in message_numbers.py and give a description (which will go to libE_stats.txt). I will change this.

Looks like its only used in uniform_or_localopt.py currently.

Reply:
I've changed this now.